packages/ui/src/stories/Colors.stories.mdx

```diff
@@ -0,0 +1,110 @@
+import { Meta } from '@storybook/addon-docs';
+
+import { ColorPalette, Title } from './_components';
+
+<Meta title="Design Tokens/Colors" />
+
+# Color Palette
+
+This design system leverages a purposeful set of color styles as the perfect starting point for any newsroom.
+
+<a
+    className="sb-unstyled text-indigo-600"
+    href="https://www.figma.com/file/46dEAasj1iEtrVQOwmMswB/00--%3E-Themes-Design-System?type=design&node-id=607-13157&mode=design&t=Ir87ygVLMpgNnBbO-4"
+    target="_blank"
+    rel="noopener noreferrer"
+>
+    See in Figma Design System
+</a>
+
+<br />
+<br />
+
+<Title
+    title="Primary Colors"
+    description="These are the main colors that make up the majority of the colors used in themes."
+/>
+
+<ColorPalette
+    title="Gray"
+    description="Gray is a neutral color and is the foundation of the color system. Almost everything — text, form fields, backgrounds, dividers — are usually gray."
+    colors={[
+        { name: 'Gray-50', hex: '#F9FAFB' },
```

Comment:
If these colors are not pulled straight from the default Tailwind config, we should define them in `tailwind.config.js`

Reply:
They are the same colors from tailwind config, used tailwind colors variables instead